README.md

```diff
@@ -110,6 +112,101 @@ The node registers itself with the gateway so the gateway will recognize message
 | app_s_key | string | yes | Application Session Key (16 bytes in hex) Used to decrypt uplink messages. Default can normally be found on the node's datasheet or box. |
 | network_s_key | string | yes | Network Session Key (16 bytes in hex) Used to decypt uplink messages. Default can normally be found on the node's datasheet or box. |
 
+## Configure the `viam:lorawan:milesight-ct101`
+
+Example OTAA node configuration:
+
+```json
+{
+  "join_type": "OTAA",
+  "dev_eui": "0123456789ABCDEF",
+  "gateways": ["gateway-1"]
+}
+```
+
+Example ABP node configuration:
+
+```json
+{
+  "join_type": "ABP",
+  "dev_addr": "01234567",
+  "gateways": ["gateway-1"]
+}
+```
+
+### Common Attributes
+
+| Name | Type | Required | Description |
+|------|------|----------|-------------|
+| join_type | string | no | Join type ("OTAA" or "ABP"). Defaults to "OTAA" |
+| uplink_interval_mins | float64 | no | Expected interval between uplink messages sent by the node. The default is **10** minutes. |
```

Comment:
let's just have one common configure section for the sensors

Reply:
@michaellee1019 do you know any markdown magic that could help us here?

Basically we have a bunch of sensors that have identical config attributes, but the value of the keys for each sensor family is different.

Reply:
Hmm. Since there are only 3 common attributes, and since inline docs are a thing now, maybe it'd be best to copy these three lines into the table for each of the models so that the inline docs can just show one table for each. I don't love the duplicated content for people looking at the whole readme and for maintenance purposes, but definitely feels like that'd make a better inline experience.

Reply:
Okay - we'll figure out if there's magic markdown to make this better another day!